lib/context.xml

```diff
@@ -0,0 +1,39 @@
+<Context>
+​
+<!-- Default set of monitored resources. If one of these changes, the    -->
+<!-- web application will be reloaded.                                   -->
+<WatchedResource>WEB-INF/web.xml</WatchedResource>
+<WatchedResource>WEB-INF/tomcat-web.xml</WatchedResource>
+<WatchedResource>${catalina.base}/conf/web.xml</WatchedResource>
+​
+<!-- Uncomment this to disable session persistence across Tomcat restarts -->
+<!--
+<Manager pathname="" />
+-->
+<Resources cachingAllowed="true" cacheMaxSize="100000" />
+<Resource name="jdbc/mat_app_tomcat"
+          auth="Container"
+          type="javax.sql.DataSource"
+          factory="org.apache.tomcat.jdbc.pool.DataSourceFactory"
+          testWhileIdle="true"
+          testOnBorrow="true"
+          testOnReturn="false"
+          validationQuery="SELECT 1"
+          validationInterval="30000"
+          timeBetweenEvictionRunsMillis="30000"
+          maxActive="100"
+          minIdle="10"
+          maxWait="10000"
+          initialSize="10"
+          removeAbandonedTimeout="60"
+          removeAbandoned="true"
+          logAbandoned="true"
+          minEvictableIdleTimeMillis="30000"
+          jmxEnabled="true"
+          jdbcInterceptors="org.apache.tomcat.jdbc.pool.interceptor.ConnectionState;
+            org.apache.tomcat.jdbc.pool.interceptor.StatementFinalizer"
+          username="root"
+          password="z0g*DR56"
```

Comment:
I'm not super familiar with docker so maybe this is supposed to be like this, but do we want someones actual root password in here?

Reply:
Just kidding, this is what I get for looking at this commit by commit. I see it's removed in the next commit. Disregard!